### DISCUSSION TITLE
Which APK is the right one?

### DISCUSSION THREAD

**Discussion by RhysSpellbinder** [ImranR98/Obtainium#1046] [Q&A] [answered]
Excuse me for my rookie question but which APK in Assets do I have to download for my google pixel 6 pro?

**Comment by juliazero** [comment, marked as answer]
![obraz](https://github.com/ImranR98/Obtainium/assets/11527277/21f58bd4-dd33-4bd5-8fad-3bc858d5133a)

**Reply by Drago-ir** [reply]
Hi,
Could you elaborate how to choose?

**Reply by ImranR98** [reply]
If your phone is recent, arm64-v8a will probably work. If not, use armabi-v7a. If unsure, you can't go wrong with the larger app-release.apk.
